python/lesson1/tutorial.md

```diff
@@ -20,17 +20,34 @@ On Windows:
 
 ## Opening the REPL
 
-Python is often developed with the aid of a *REPL*, or *Read-Eval-Print-Loop*. The REPL is a way of getting immediate feedback as you work.
+Python is often developed with the aid of a *REPL*, or *Read-Eval-Print-Loop*. The REPL is a way of getting immediate feedback as you work. Python's REPL is called **IDLE**.
 
-Python's REPL is called **IDLE**.
+It's very important to find what Python version you are working with.
 
-On Windows, you can run it by clicking on the shortcut in your Start menu or Start screen named "IDLE (Python GUI)".
+On Windows, open `cmd` program and type (without the `$` sign):
 
-On Mac OS, open the Terminal and type "python" to start the REPL. At the top, you'll see something similar (but probably not identical) to this:
+    $ python --version
 
-    Python 2.7.6 (default, Sep  9 2014, 15:04:36)
+On Mac OS / GNU/Linux, open the Terminal and type (without the `$` sign)
 
-If you see it, make sure you're running Python 2, not Python 3, otherwise parts of this exercise will not make sense. Once you've checked that, we're ready to move on to writing Python.
+    $ python --version
```

Comment:
As this command is the same regardless of what machine you're using do you think they need splitting up. Maybe combine lines 27 and 31?